edwin/client/static/js/stores/BugStore.js

```diff
@@ -102,6 +114,9 @@ function augmentBug(bug) {
   // Parse the whiteboard field
   bug = bug.set('whiteboardParsed', Immutable.fromJS(whiteboardData.parse(bug.get('whiteboard', ''))));
 
+  // Get needinfo flags for this bug.
+  bug = bug.set('needinfo', Immutable.fromJS(getNeedinfo(bug)));
```

Comment:
`getNeedInfo` calls `.toJS()`, but then you immediatley re-wrap it `Immutable.fromJS`. It would probably better to remove both calls.